Samples/azure-storage/Azure.Java.Fluent/implementation/StorageAccountInner.java

```diff
@@ -5,6 +5,7 @@
 
 import petstore.StorageAccountProperties;
 import com.microsoft.azure.Resource;
+import com.microsoft.azure.Resource;
```

Comment:
It's a bug in Java codegen. I've already fixed it locally. I'll make sure the samples are regenerated when I send a PR. Thanks for catching this!